### PR TITLE
feat: add aggregated /chains/status endpoint

### PR DESCRIPTION
**Pull request by pitasi**
This PR adds `/chains/status` which aggregates all the `/chain/XXX/status` calls into one.

The list of the returned chains only contains **enabled** chains. This information can be used by the frontend to establish whether a chain is enabled or not.

Close #715.

### CODE REVIEW

File: api/chains/chains.go [hunk 1]
```diff
@@ -1240,3 +1240,40 @@ func getAPR(c *gin.Context, sdkServiceClients sdkservice.SDKServiceClients) stri
...
 }
+
+// GetChainsStatuses returns the status of all the enabled chains.
+// @Summary Gets status for all enabled chains.
+// @Tags Chain
+// @ID statuses
+// @Description Gets status for all enabled chains.
+// @Produce json
+// @Success 200 {object} ChainsStatusesResponse
+// @Failure 500,403 {object} apierrors.UserFacingError
```
Comment (DeshErBojhaa) [inline api/chains/chains.go]: Do we return 403?

File: api/chains/models.go [hunk 2]
```diff
@@ -170,3 +170,17 @@ type Pagination struct {
...
+
+type ChainsStatusesResponse struct {
+	Chains map[string]ChainStatus `json:"chains"`
+}
+
+func NewChainsStatusesResponse() ChainsStatusesResponse {
+	return ChainsStatusesResponse{
+		Chains: make(map[string]ChainStatus),
+	}
+}
```
Comment (DeshErBojhaa) [inline api/chains/models.go]:
```suggestion
func NewChainsStatusesResponse(sz int) ChainsStatusesResponse {
	return ChainsStatusesResponse{
		Chains: make(map[string]ChainStatus, 0, sz),
	}
}
```
then on `GetChainsStatuses`  

```go
res := NewChainsStatusesResponse(len(status))
```

**Review by DeshErBojhaa — CHANGES_REQUESTED** [review]

File: api/chains/chains.go (hunk 1 above)
Comment (pitasi) [reply api/chains/chains.go]: nope!, removed

File: api/chains/models.go (hunk 2 above)
Comment (pitasi) [reply api/chains/models.go]: It's a map so I omitted the "0" but added a size!

Comment (DeshErBojhaa) [reply api/chains/models.go]: opps my bad :p

**Review by DeshErBojhaa — APPROVED** [review]
🎊

**Review by akhilkumarpilli — APPROVED** [review]

**Pull request merged**